contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -3601,3 +3601,10 @@ AS SELECT
 FROM sys.objects so
 WHERE FALSE;
 GRANT SELECT ON sys.sequences TO PUBLIC;
+
+CREATE OR REPLACE VIEW sys.dm_os_sys_info 
```

Comment:
according to TSQL doc, this view is supposed to contain many more columns

Reply:
Yes other are related to hardware and os informations. For supporting Proxy this below added columns are sufficient. I was thinking of add these fields now and add the other fields as and when required.

Reply:
I think we should add all columns right now. We can put dummy values for the columns that do not apply to us and we can update those later if we ever find suitable values for them. Changing values in upgrade scripts is very straightforward but adding columns, not so much.

Reply:
+1